fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/block/FabricMaterialBuilder.java

```diff
@@ -43,8 +43,9 @@ public FabricMaterialBuilder pistonBehavior(PistonBehavior behavior) {
 		return this;
 	}
 
+	@Override
 	public FabricMaterialBuilder lightPassesThrough() {
```

Comment:
Does this keep binary compat? The vanilla method is package-private so this didn't override it before, meaning it got the name `lightPassesThrough` in the obf jar (I confirmed this). Now it'll probably result in `method_15812` in the obf jar.

Reply:
I actually posed this question in the PR description, but it seems no one noticed that...
I'm not sure, and if it doesn't then I don't know how to go about it short of straight up removing the access widener.

Reply:
Good spot, yes I think will will cause issues. Like you said I think removing the AW is the only option?

Reply:
Removed the widener for that method. Remind me to add it back in for the 1.20 cycle.

Reply:
Great, I think that sounds reasonable. 👍